anyway/constants.py

```diff
@@ -37,6 +37,9 @@ class Constants(object):
     # other global constants (python only)
     DEFAULT_NUMBER_OF_YEARS_AGO = 5
 
+    # years ago to store in cache
+    INFOGRAPHICS_CACHE_YEARS_AGO = [3, 5]
```

Comment:
can we add 1 and 8?

Reply:
Sure, do you want me to add? Note that it will increase the cache build time, probably to ~12 hours.

Reply:
Yes, please add. No problem regarding the cache build since as far as I understand from you, it's not a down time right?

Reply:
Correct.
in the future we may want to change a little bit the cache build so in case it is interrupted not all the time will be wasted.